recipes/ipycache/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: ipycache
+  version: {{ version }}
+
+source:
+  fn: ipycache-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/i/ipycache/ipycache-{{ version }}.zip
+  md5: c78c6094ffab45b0e33c2f556705fa11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - ipython
+
+test:  
+  commands:
+    - git clone --depth=50 https://github.com/rossant/ipycache.git source
+    - cd source 
+    - nosetests 
+    - python ipynb_runner.py -v -s examples/example.ipynb  # [not win]
+    - python ipynb_runner.py -v -s examples/example_outputs.ipynb  # [not win]
+    - cd ..
+  
+  requires:
+    - nose
+    - numpy
+    - matplotlib
+    - ipython
+    - nbformat
+    - ipykernel
+
+about:
+  home: http://packages.python.org/ipycache
+  license: BSD License
```

Comment:
@jochym the alternative is to write BSD-like and add the actual license file here using the key license_file: fname